### PR TITLE
update feature posts

### PR DESCRIPTION
**Pull request by caovv62**
create feature posts of blog

### CODE REVIEW

File: resources/css/index.css [hunk 1]
```diff
@@ -117,12 +117,72 @@
...
+
+#feature-posts h1{
+  color:black;
+  font-size: 30px;
+  font-family: Arial, Helvetica, sans-serif;
+  padding-top: 10px;
+}
+#feature-posts .card-list {
+  width: 100%;
+  /*display: flex;*/
```
Comment (ngvuthanhnhan) [inline resources/css/index.css:134]: Remove comment if it is not necessary.

Comment (ngvuthanhnhan) [reply resources/css/index.css:134]: Fix similar error in your PR.

File: index.html [hunk 2]
```diff
@@ -37,7 +39,16 @@
     <div class="posts-container">
       <div id="feature-posts">
         <h1>Featured Posts</h1>
-        Should we feature some popular posts here, make it a slider?
+        <hr>
+          <div class="card-list"></div>
+          <div class="card-list"></div>
+          <div class="card-list"></div>
+          <br>
+          <div style="text-align:center">
```
Comment (ngvuthanhnhan) [inline index.html:47]: According to our coding convention, style shouldn't be inline a tag. 
Name your div as class or id, define its style in .css file.

File: index.html [hunk 3]
```diff
@@ -79,8 +90,10 @@ <h3 class="title">Subscribe Me</h3>
         </div>
       </div>
     </div>
+    <script src="resources/js/feature_post.js"></script>
+    <script src="resources/js/responsive.js"></script>
   </body>
-
   <script src="resources/js/responsive.js"></script>
   <script src="resources/js/cover.js"></script>
+  <script src="resources/js/feature_post.js"></script>
```
Comment (ngvuthanhnhan) [inline index.html:98]: Wrong naming convention: https://www.dropbox.com/scl/fi/k4a5rv4kcgt234z6no0ih/_-Coding-Convention.paper?dl=0&rlkey=ka3xx7whfsjq61scx8smcesya

File: resources/js/feature_post.js [hunk 4]
```diff
@@ -0,0 +1,188 @@
+const feature_post = [
```
Comment (ngvuthanhnhan) [inline resources/js/feature_post.js:1]: Viết thế này, ví dụ như mình có 5, 7 hay số feature posts không chia hết cho 3 thì sao anh?

File: resources/js/feature_post.js [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+    {
+      image: 'resources/images/abramovich.jpg',
+      title: '9. Super ultra hot videos Super ultra hot videos Super ultra hot videos',
+      view: ' 10.000',
+    },
+  ];
+  
+  var listcard = document.getElementById('feature-posts').getElementsByClassName('card-list')[2];
+    for (var i = 0; i < feature_post_2.length; i++) {
+      var icon = document.createElement('I');
```
Comment (ngvuthanhnhan) [inline resources/js/feature_post.js:131]: Các thẻ i, div, img viết hoa lên hết làm gì nhỉ?

**Comment by ngvuthanhnhan** [comment]
Screen Shot 2020-06-26 at 8 33 46 PM
This is not what is called "slider". Also there is no transition between pages, and when clicking a dot to change pages, old (current) dot disappear.

File: resources/css/card.css [hunk 6]
```diff
@@ -0,0 +1,29 @@
+.card-container {
+    width: 300px;
+    padding-left: -10px;
+  }
+  
+  .card-container .card {
```
Comment (ngoc-nguyenminh) [inline resources/css/card.css:6]: No tab before element name

File: resources/css/card.css [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+    width: 100%;
+    height: 100%;
+    background-color: #fff;
```
Comment (ngoc-nguyenminh) [inline resources/css/card.css:9]: 6 bit-hex color pls

File: resources/css/responsive.css [hunk 8]
```diff
@@ -110,4 +110,12 @@
     margin-top: 20px;
     width: 100%;
   }
-}
+/* feature-post */
+@media screen and (min-width: 320px) {
+  .card-list .card-container {
```
Comment (ngoc-nguyenminh) [inline resources/css/responsive.css:115]: No tab

File: resources/js/feature_post.js [hunk 9]
```diff
@@ -0,0 +1,188 @@
...
+  for (i = 0; i < slides.length; i++) {
+      slides[i].style.display = "none";
+  }
+  for (i = 0; i < dots.length; i++) {
+      dots[i].className = dots[i].className.replace(" active", "");
+  }
+  slides[slideIndex-1].style.display = "flex";
+  dots[slideIndex-1].className += "active";
+}
+
```
Comment (ngoc-nguyenminh) [inline resources/js/feature_post.js:188]: No newline
And if you have time, please refactor your JS implementation, it's too complicated for a slider
Check again your slider UI. It has bugs. Next time. please check thoroughly before making a Pull Request

**Pull request closed without merging**